src/utilities/focus.ts

```diff
@@ -154,8 +154,9 @@ export function isElementTabbable(element: HTMLElement): boolean {
     !!element &&
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
-      (element.tagName === 'INPUT' && !(element as HTMLButtonElement).disabled) ||
-      (element.getAttribute && element.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true')));
+      (element.tagName === 'INPUT' && !(element as HTMLInputElement).disabled) ||
+      (element.getAttribute && element.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true')) ||
+      (element.getAttribute && element.getAttribute('tabindex') === '0'));
```

Comment:
A better change would be to render the Slider as a button; this works a lot better since buttons natively have a disabled attribute. I believe this will address your issue a little more effectively.

https://github.com/OfficeDev/office-ui-fabric-react/pull/253


I've created a PR here:

https://github.com/OfficeDev/office-ui-fabric-react/pull/253